src/js/ko-components/view-models/orifice-calculator.coffee

```diff
@@ -62,3 +66,22 @@ class OPL.KoComponents.ViewModels.OrificeCalculator
 
     @compressibilityCorrection = ko.observableArray _.values OPL.OrificeCalculator.Config.Dictionaries.CompressibilityCorrection
     @chosenCompressibilityCorrection  = ko.observable OPL.OrificeCalculator.Config.Dictionaries.CompressibilityCorrection.none
+    @compressibilityCorrectionValue = ko.observable(1)
+    @betaRatio = ko.computed =>
+      @orificeBoreDiameter() / @selectedPipeID()
+
+    @velocityOfApproach = ko.computed =>
+      Math.ceil((1 / Math.sqrt 1 - @betaRatio() ** 4) * 10 ** 2) / 10 ** 2
```

Comment:
Please add parenthesis to math.sqrt to emphasize which params should be in the SQRT.
`Math.ceil((1 / Math.sqrt(1 - @betaRatio() ** 4)) * 10 ** 2) / 10 ** 2****`